src/chat.cpp

```diff
@@ -564,8 +563,7 @@ ChatChannel* Chat::getChannel(const Player& player, uint16_t channelId)
 		}
 
 		case CHANNEL_PARTY: {
-			Party* party = player.getParty();
-			if (party) {
+			if (const auto& party = player.getParty()) {
```

Comment:
It's been my experience that declaring a variable and checking it inside the if statement causes crashes. I'm sure this isn't true in all cases, but I'm just stating what has been my experience so far.

Reply:
How so? It's equivalent other than the scoping (which would cause compilation errors, not crashes)

Would be interesting to see a form of reproduction though